xbmc/utils/URIUtils.cpp

```diff
@@ -777,6 +777,24 @@ bool URIUtils::IsDVD(const std::string& strFile)
   return false;
 }
 
+bool URIUtils::IsDVDFile(const std::string& strFile)
+{
+  std::string strFileLow = strFile;
+  StringUtils::ToLower(strFileLow);
+  if (strFileLow.find("video_ts.ifo") != std::string::npos)
+    return true;
+  return false;
```

Comment:
`return (fileLow.find("video_ts.ifo") != std::string::npos));`

Reply:
The implementation is not correct, as it will also return true if we pass in something containing "video_ts.ifo" in the middle of the path, like "/foo/video_ts.ifo/bar/baz.txt". It should look whether the passed in file ends (!) with "/video_ts.ifo" or if it does not start with a "/", equals "video_ts.ifo".

Reply:
Done